tools/sdk-readme-table-generator/TableGenerator/Program.fs

```diff
@@ -8,7 +8,10 @@ open TableGenerator.Table
 let inputBranches =
         [ { GitBranchName = "main"
             DisplayName = "main<br>(9.0.x&nbsp;Runtime)"
-            AkaMsChannel = Some("9.0.1xx/daily") }
+            AkaMsChannel = Some("9.0.1xx/daily") }'
```

Comment:
Is that extra ' a typo?  Also, you have to manually update the readme file with the output of running the tablegenerator with the changes.

Reply:
I don't even know how I put a typo on a line I didn't intentionally edit, but yes; thanks for catching that.

And now I've run my first F# script!